src/orchestrator-kubernetes/src/lib.rs

```diff
@@ -353,12 +353,12 @@ impl k8s_openapi::Metadata for MetricValueList {
 impl NamespacedKubernetesOrchestrator {
     /// Return a `ListParams` instance that limits results to the namespace
     /// assigned to this orchestrator.
-    fn list_pod_params(&self) -> ListParams {
+    fn list_pod_params(&self) -> watcher::Config {
         let ns_selector = format!(
             "environmentd.materialize.cloud/namespace={}",
             self.namespace
         );
-        ListParams::default().labels(&ns_selector)
+        watcher::Config::default().labels(&ns_selector)
```

Comment:
Should we just rename this method, since it looks like it's only used in a `watch()` invocation anyway?